pkg/operator/ceph/object/admin_test.go

```diff
@@ -256,7 +256,7 @@ func TestCommitConfigChanges(t *testing.T) {
 			},
 			expectCommands{
 				periodUpdate: true,
-				periodCommit: false,
+				periodCommit: true,
```

Comment:
@BlaineEXE By updating to go-ceph v0.18, the period is now being committed. Does this sound expected for this test?

Reply:
I don't think that is expected. Let me pull these changes locally and poke around.

Reply:
I find it more likely that possibly go-cmp has changed from v0.5.8 to v0.5.9

Reply:
Any concern with merging this for now?

Reply:
Yes. This will mean the RGW period gets updated on every reconcile. IIRC, the behavior to commit only if changes will be made was done so as to not cause any hiccups in RGW availability unless necessary.